.github/workflows/tests.yml

```diff
@@ -9,16 +9,24 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
+      fail-fast: false
       matrix:
+        python-version: ['3.8', '3.9', '3.10',]
         os: [ubuntu-latest, windows-latest, macos-latest]
+        experimental: [false,]
+        include:
+          - python-version: '3.11'
+            os: ubuntu-latest
+            experimental: true
```

Comment:
**suggestion:** I learn toward just removing 3.11 for now.

Reply:
OK, we can do that (nontheless, check shows as passing in the summary view https://github.com/wfondrie/mokapot/actions). LMK if this is the last call

Reply:
Yes, at least for now.